### DISCUSSION TITLE
最新版本无法联网

### DISCUSSION THREAD

**Discussion by Kanbe-Kotori** [1715173329/UnblockNeteaseMusic#503] [Q&A]
测试的时候显示代理可用，但是什么也加载不出来。。

**Comment by 1715173329** [comment]
- 平台
- 版本
- 参数
- 日志

**Comment by Kanbe-Kotori** [comment]
平台：windows
版本：网易云版本是2.9.7，代理是刚刚download zip下来的，2022.1.29更新的版本
参数：node app.js -p 23333
日志：没有找到.log文件，控制台啥也没有

**Comment by Kanbe-Kotori** [comment]
测试的时候显示代理可用，然后重启网易云应用之后显示“网络连接失败请重试”
今天未更新网易云版本之前可用，我听着听着歌他给我自动更新了，然后就连不上了。
实测编译好的rc4exe版本可以运行，但直接git clone下来的不行。

**Comment by 1715173329** [comment]
代理检测需要反着看，提示“代理不可用”才代表有效。
新版本需要监听 https 端口，如 -p 23333:23334
